client/src/test/java/org/spine3/base/EventsShould.java

```diff
@@ -244,6 +245,17 @@ public void return_optional_absent_if_no_needed_event_enrichment_when_getting_on
                           .isPresent());
     }
 
+    @Test
+    public void pass_the_null_tolerance_check() {
+        final NullToleranceTest nullToleranceTest = NullToleranceTest.newBuilder()
+                                                                     .setClass(Events.class)
+                                                                     .addDefaultValue(newArrayList())
```

Comment:
Instead let's provide the default values for the collections in the `NullToleranceTest` itself.